cli/src/cli_util.rs

```diff
@@ -2085,9 +2100,9 @@ See https://jj-vcs.github.io/jj/latest/working-copy/#stale-working-copy \
         // don't leave the working copy in a stale state.
         if self.may_update_working_copy {
             if let Some(new_commit) = &maybe_new_wc_commit {
-                let locked_ws = self.workspace.start_working_copy_mutation()?;
+                let locked = locked_ws.expect("locked_ws should be held if updating WC");
```

Comment:
nit: Call this new variable `locked_ws` too? We frequently redefine variables like that when they have similar types (and we do it on line 2116)